ISSUE_TEMPLATE/testing.md

```diff
@@ -0,0 +1,82 @@
+# Prerequisites
+
+You will need:
+- access to the official CentOS CI fedora-coreos namespace
+- access to the AWS S3 fcos-builds bucket
+- the following packages: `awscli gnupg2 git`
+- [`fedora-coreos-stream-generator`](https://github.com/coreos/fedora-coreos-stream-generator/)
+- your GPG key linked to your FAS account
+
+# Promote testing-devel changes
+
+If you haven't already, check out both [`fedora-coreos-releng-automation`](https://github.com/coreos/fedora-coreos-releng-automation) and [`fedora-coreos-config`](https://github.com/coreos/fedora-coreos-config). You'll also want to fork both of them on GitHub.
```

Comment:
I'm in favor of moving everything above here into a separate prerequisites doc in this repo, and linking to that doc from the template.  That keeps the checklist free of verbiage that only applies once.